timelock-agent/src/main/java/com/palantir/timelock/config/TimeLockRuntimeConfiguration.java

```diff
@@ -44,6 +48,24 @@ public long slowLockLogTriggerMillis() {
         return 10000;
     }
 
+    /**
+     * The namespace and token credentials the server should use to authenticate basic service clients.
+     */
+    @JsonProperty("service-auth-credentials")
+    @Value.Default
+    public List<TimelockClientCredentials> serviceAuthCredentials() {
+        return ImmutableList.of();
```

Comment:
this doesn't need the @Value.Default (neither does the other one) - it's inferred for List types.